be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/ElectionHandler.scala

```diff
@@ -71,15 +75,83 @@ class ElectionHandler(dbRef: => AskableActorRef) extends MessageHandler {
 
   def handleCastVoteElection(rpcMessage: JsonRpcRequest): GraphMessage = {
     // no need to propagate here, hence the use of dbAskWrite
-    val ask: Future[GraphMessage] = dbAskWrite(rpcMessage)
+    val ask: Future[GraphMessage] = dbAskWritePropagate(rpcMessage)
```

Comment:
Yes. We need to send it back to the frontend so that they can compute `registered_votes` of the EndElection message. We already discussed that with the evoting team, so yes we need to propagate the message.